src/nccl_ofi_interface_nvidia.c

```diff
@@ -32,8 +32,36 @@ static ncclResult_t getProperties_v8(int dev_id, ncclNetProperties_v8_t* props)
 	 * support this model (since NCCL maintains a per-domain registration
 	 * cache which requires (domain-)global registrations.
 	 */
-	if (ofi_properties.mr_scope == NCCL_OFI_MR_SCOPE_DOMAIN)
-		props->regIsGlobal = 1;
+	if (ofi_properties.mr_scope == NCCL_OFI_MR_SCOPE_DOMAIN) {
+		//props->regIsGlobal = 1;
```

Comment:
Don't comment out code.  Definitely don't comment out code and stick a giant comment between it and the new code, so that this code never gets cleaned up.

Reply:
Removed